client/my-sites/checkout/controller.jsx

```diff
@@ -75,7 +75,6 @@ export default {
 			<CheckoutData>
 				<Checkout
 					product={ product }
-					productsList={ productsList }
```

Comment:
Uhm, we pass `productsList` to `CheckoutThankYouComponent` where I can't find it used anywhere and to `GsuiteNudge` where it seems it is used. Should we reduxify `GsuiteNudge` too and get rid of the `productsFactory`?

Reply:
I will probably clean checkout after this, but since it's a nasty bug, wanted to get rid of it first.